modules/installation-azure-confidential-vms.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-confidential-vms_{context}"]
+= Enabling Confidential VMs
```

Comment:
It's decided to be a TP in 4.14, do we need to highlight this in doc, or is it enough to mention only in RN?

Reply:
I can confirm that it's TP in 4.14.

Reply:
I've added the tech preview snippet to the docs.